types/gestalt/index.d.ts

```diff
@@ -1360,12 +1365,12 @@ export interface TextAreaProps {
 export interface TextFieldProps {
     id: string;
     onChange: (args: { event: React.SyntheticEvent<HTMLInputElement>; value: string }) => void;
-    autoComplete?: 'current-password' | 'on' | 'off' | 'username' | 'new-password';
+    autoComplete?: 'current-password' | 'on' | 'off' | 'username' | 'new-password' | 'email';
     /**
      * @default false
      */
     disabled?: boolean;
-    errorMessage?: string;
+    errorMessage?: React.ReactNode | string;
```

Comment:
nit: `React.ReactNode` includes `string` type so `React.ReactNode | string` is redundant.